daisy_workflows/image_build/windows/post_install.ps1

```diff
@@ -182,6 +182,18 @@ function Install-WindowsUpdates {
     }
   }
 
+  # Windows 11 may get stuck installing KB5007651. $pn for Windows 11 reports as Win 10 Enterprise.
```

Comment:
Windows 11 has a build number of 10.0.22000
https://en.wikipedia.org/wiki/Windows_11_version_history

Windows 10 21h2 has a build number of 10.0.19044.2006
https://en.wikipedia.org/wiki/Windows_10_version_history

So for Windows 11 detection logic:

$productMajorVersion = (Get-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion' -Name CurrentMajorVersionNumber).CurrentMajorVersionNumber
$productMinorVersion = (Get-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion' -Name CurrentMinorVersionNumber).CurrentMinorVersionNumber
$productBuildNumber = (Get-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion' -Name CurrentBuildNumber).CurrentBuildNumber

Windows 11
($productMajorVersion -eq 10 -and $productMinorVersion -eq 0 -and $productBuildNumber -ge 22000)

Windows 10 
($productMajorVersion -eq 10 -and $productMinorVersion -eq 0 -and $productBuildNumber -le 22000 -and $productBuildNumber -ge 10240)

Reply:
I added this to run only if there is 1 update remaining so we don't end up running this more than it needs to be. Question for a potential future PR is if we want to update so that we use these values versus product name for other instances like the Win7 skip. Seems more reliable.